assembl/static2/js/app/components/form/utils.js

```diff
@@ -49,6 +57,25 @@ export function convertEntriesToI18nRichText(entries: RichTextLangstringEntries)
   return convertEntriesToI18nValue(convertEntriesToEditorState(entries));
 }
 
+export function convertISO8601StringToDateTime(_entry: string): DatePickerInput {
+  if (_entry) {
+    const t = moment(_entry, moment.ISO_8601).utc();
+    if (t._isValid) { // eslint-disable-line no-underscore-dangle
```

Comment:
Why do you access a private api `t._isValid`? I modified it in https://github.com/assembl/assembl/commit/e259dbb57f83c7d030fd7f1d7ab0bf349fbde75f to use `t.isValid()`